devito/operator/operator.py

```diff
@@ -386,6 +386,9 @@ def _lower_iet(cls, stree, profiler, **kwargs):
         graph = Graph(iet)
         graph = cls._specialize_iet(graph, **kwargs)
 
+        # Add MIN/MAX defines to the IET
+        add_minmax(graph)
```

Comment:
it's not definitely adding a new pass that we'll get min/max to pop up in the generated code...